video/users/client-initialization-rooms/client-initialization.js

```diff
@@ -1,10 +1,9 @@
 $.getJSON('/token', function(data) {
 
   // Create a Video Client and connect to Twilio's backend
-  var videoClient = new Twilio.Video.Client(data.token);
-  videoClient.connect({ to: 'my-room'}).then(function(room) {
+   Twilio.Video.connect('$TOKEN', { name: 'my-room'}).then(function(room) {
       console.log('Connected to Twilio!');
   }, function (error) {
       console.log('Could not connect to Twilio: ' + error.message);
   });
-});
+});
```

Comment:
@ktoraskartwilio I don't see the need for this snippet any longer, since there is no `Client` now.